backend/app/services/llm/providers/gasr.py

```
@@ -0,0 +1,212 @@
+from google.cloud import speech_v2
+from google.cloud.speech_v2 import types
+import google.auth
+
+import base64
+from typing import Any, Dict, Optional, Tuple
```

Ruff (0.14.14)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)